### PR TITLE
Readme Fix

### PR DESCRIPTION
**Pull request by francocorreasosa**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Ever wanted to share a certain directory on your network by running just a command? Then this module is exactly what you're looking for: It provides a neat interface for listing the directory's contents and switching into sub folders.
 
-In addition, it's also awesome when it comes to serving static sites. If a directory contains an `index.html`, `serve` will automatically render it instead of serving the file's content as plaintext.
+In addition, it's also awesome when it comes to serving static sites. If a directory contains an `index.html`, `serve` will automatically render it instead of serving directory's content, and will serve any `.html` file as a rendered page instead of file's content as plaintext.
```
Comment (timneutkens) [inline readme.md]: `directory's content` -> `directory contents`

Comment (timneutkens) [inline readme.md]: @francocorreasosa speeeeeed!

Comment (francocorreasosa) [inline readme.md]: Sorry, non-native english speaker here 😄

**Comment by leo** [comment]
🙏

**Pull request merged**